docs/README.md

```diff
@@ -1,6 +1,35 @@
 
-## Delorean
+# Delorean
 
-  - [1. Early Warning System](delorean/early-warning-system.md)
-    - [1.1 Add New Product](delorean/add-new-product.md)
-  - [2. Jenkins Configuration](delorean/configuring-jenkins.md)
+## Table of Contents
+  - [1. Overview](#overview)
+  - [2. Early Warning System](#early-warning-system)
+    - [2.1 Delorean Jobs](#delorean-jobs)
+    - [2.2 Adding New Products](#adding-new-products)
+  - [3. Cluster Management](#cluster-management)
+  - [4. Automated Tests](#automated-tests)
+  - [5. Jenkins Configuration](#jenkins-configuration)
+
+## Overview
+Multiple products are installed as part of the Integreatly environment. It is important to move quickly to the new releases of these products to enable proof of concepts and ensure any new managed environment is always using the latest versions. The Delorean suite of jobs provides an automated way of discovering, updating and testing new versions of these products.
+
+## Early Warning System
+These products are configured to integrate with each other, therefore, it is important to receive early warnings of changes to the installation, configuration and functionality of each of these products.
+
+### Service Discovery
+Delorean provides service discovery for each of the products installed in the Integreatly environment. For further information on the different types of Delorean jobs, please go [here](delorean/early-warning-system.md).
```

Comment:
Service discovery is not really right, more like release discovery or version discovery or something